04 Abstraction Exercises Part 1/Enemy-Application/src/main/java/com/shooting/game/enemy/Enemy.java

```diff
@@ -0,0 +1,15 @@
+package com.shooting.game.enemy;
+
+public class Enemy {
+    private int attackPoints;
+    private int hitPoints;
```

Comment:
make all instances of the enemy have the same attackpoints and hitpoints. meaning everytime an enemy instance spawn, it has the same AP and HP

Reply:
Done na po sir, though not sure if tama po ginawa